### PR TITLE
details about cookies being used.

### PR DESCRIPTION
**Pull request by Faizi52996**
### Context
cookies documentation

### Change proposed in this pull request
n/a

### Guidance to review 
n/a
### Checklist
~~- [ ] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)~~
~~- [ ] Your code builds clean without any errors or warnings~~
~~- [ ] You have run all unit/integration tests and they pass~~
- [ ] Your branch has been rebased onto main
~~- [ ] You have tested by running locally~~

### CODE REVIEW

**Review by jrabbott — COMMENTED** [review]
Please update index page in repo readme

File: documentation/developers/Cookies.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿# Cookie Documentation
+
```
Comment (jrabbott) [inline documentation/developers/Cookies.md:2]: Please add brief intro re: purpose of this document

File: documentation/developers/Cookies.md
Comment (jrabbott) [inline documentation/developers/Cookies.md:1]: Could you add a table add the intro with headings;

Cookie / Purpose / Expires after

Keep the more details sections. Table helps to provide a quick summary

Comment (jrabbott) [inline documentation/developers/Cookies.md:1]: Please update index page in repo readme

File: documentation/developers/Cookies.md (hunk 1 above)
Comment (Faizi52996) [reply documentation/developers/Cookies.md]: done.

File: documentation/developers/Cookies.md
Comment (Faizi52996) [reply documentation/developers/Cookies.md:1]: ahh yes added it now.

Comment (Faizi52996) [reply documentation/developers/Cookies.md:1]: ahh yes added it now.

**Review by jrabbott — APPROVED** [review]

**Pull request merged**